src/storage/v2/inmemory/storage.cpp

```diff
@@ -769,34 +765,31 @@ utils::BasicResult<StorageManipulationError, void> InMemoryStorage::InMemoryAcce
         // so the Wal files are consistent
```

Comment:
For checking `if(!unique_constrant_violation)` prefer early return for removing one indentation

Reply:
Outside the scope of PR. Don't want to change ATM, maybe a refactor for another PR.